google-cloud-vision-v1/Rakefile

```diff
@@ -161,7 +161,7 @@ end
 
 task :cloudrad do
   ENV["CLOUDRAD_GEM_NAME"] = "google-cloud-vision-v1"
-  sh "bundle exec yard doc --fail-on-warning --yardopts .yardopts-cloudrad"
+  sh "bundle exec yard doc --fail-on-warning --yardopts .yardopts-cloudrad --no-progress"
```

Comment:
This is going to get overwritten by the generator. Do we need to update the generator to do this for all clients?

Reply:
woops that was a debug thing. removing